dashboard/config/application.rb

```diff
@@ -100,8 +100,6 @@ class Application < Rails::Application
       jquery.handsontable.full.css
       jquery.handsontable.full.js
     )
-    config.react.variant = :development
```

Comment:
If I run `NODE_ENV=production ./build-js.js` it reduces the size of **build/js/code-studio.js** by about 18KB.

I can achieve the same effect by adding `NODE_ENV: 'production'` to the environment hash we pass to child_process.execSync.  I think it's six one way, half-dozen the other.  Maybe we just make sure our CI scripts invoke this script with NODE_ENV set to production, because we want staging/test building shippable output?